src/Microsoft.Diagnostics.Monitoring.RestServer/IDiagnosticServices.cs

```diff
@@ -7,15 +7,14 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Diagnostics.NETCore.Client;
 
 namespace Microsoft.Diagnostics.Monitoring.RestServer
 {
     /// <summary>
     /// Set of services provided by the monitoring tool. These are consumed by
     /// the REST Api.
     /// </summary>
-    public interface IDiagnosticServices : IDisposable
+    internal interface IDiagnosticServices : IDisposable
```

Comment:
While I'm ok with this, I also think it's ok to leave a handful of public interfaces inside the RestServer assembly. I don't think these would be perceived as a public api but rather dependency injection requirements for the Rest server itself.

Reply:
This is a downstream effect of me adding IEndpointInfo property to the IProcessInfo interface. Everything that exposed IProcessInfo had to become internal because it is internal.